arches/app/models/graph.py

```diff
@@ -965,6 +965,11 @@ def update_node(self, node):
         """
         node["nodeid"] = uuid.UUID(str(node.get("nodeid")))
         old_node = self.nodes.pop(node["nodeid"])
+
+        for sibling_node in self.get_sibling_nodes(old_node):
```

Comment:
It seems like this same logic is used in validate_node_name which gets called in graph.validate.

Reply:
I'm wondering if we are somehow skipping the validate method or if errors raised in validate are not getting passed back to the frontend properly.